docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -19,8 +19,7 @@ To deploy Elastic Serverless Forwarder, you have to:
 [discrete]
 [[aws-serverless-forwarder-deploy-prereq]]
 == Prerequisites
-This documentation assumes you have some familiarity with {aws} services and you have correctly created and configured the necessary {aws} objects. For example, if you want to use an Amazon S3 (via SQS event notifications) input then you must ensure that you have enabled AWS VPC flow logs to be sent to that bucket, and created an SQS queue to receive those logs. For more information, refer to the relevant https://docs.aws.amazon.com/[{aws} docs].
```

Comment:
I am removing this because it does not read right. There are other forms to get logs in S3.